common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -4,7 +4,8 @@
 @import templates.inlineJS.blocking.js._
 
 <!--[if lt IE 9]>
-    <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
+    <script src="@Static("javascripts/ie8polyfills.js")"></script>
+    <script>@Html(polyfills().body)</script>
```

Comment:
@sndrs - yeah, but you could at least place `polyfills()` straight after the conditional comment. It's all render-blocking code in any case, so the order shouldn't change when the browser will get to paint anything.